modules/client-common/src/main/java/org/apache/ignite/client/proto/TuplePart.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.proto;
+
+/**
+ * Defines tuple part to write or read - key, value, or both.
+ */
+public enum TuplePart {
+    KEY,
+    VAL,
+    KEY_AND_VAL
```

Comment:
What about 'contains' operation?  It don't require any key or value to be returned.
It's ok if you plan to optimize it later.

Reply:
Maybe we should add a separate ticket for this part.

Reply:
Good catch, added `ClientOp.TUPLE_CONTAINS_KEY`.